PRFCS/prfc-1.md

```diff
@@ -51,50 +54,50 @@ Returns information about the Token implemented by this contract.
 fn allowance(owner: PublicAddress, spender: PublicAddress) -> u64
 ```
 
-Returns the number of tokens currently in the spender's allowance that they can spend on the ‘owner’s’ behalf.
+Returns the amount of tokens that the `spender` can currently spend on behalf of the `owner`.
 
 ### balance_of
 ```rust
 fn balance_of(address: PublicAddress) -> u64
 ```
 
-Queries the balance for an owner account 'address'.
+Queries the amount of tokens in an account identified by `address`.
 
 
 ## Required Calls
 ---
 
 ### transfer
 ```rust
-fn transfer(to_address: Option<Public_Address>, amount: u64)
+fn transfer(to_address: Option<PublicAddress>, amount: u64)
 ```
 
-`transfer` transfers tokens to an address from the account identified by `txn.signer`. If `to_address` is None, this burns the amount.
+Transfers tokens to an address from the account identified by `calling_account`. If `to_address` is None, this burns the amount.
```

Comment:
Seems better this way:
Transfers tokens to an account identified by `to_address` from the  `calling_account`. If `to_address` is None, this burns the amount.